ui/src/app/pages/runtimes-list.tsx

```diff
@@ -132,7 +137,8 @@ export const RuntimesList = fp.flow(
             <Environments
               {...{
                 nav: {
-                  getLink: (_, { namespace }) => `/workspaces/${namespace}`,
+                  getLink: (_, { namespace, name }) =>
+                    `/workspaces/${namespace}/${stringToSlug(name)}/data`,
```

Comment:
While `slug(name)` should always produce an equal string, the actual field used for routing is workspace `id` - let's continue to use that so we don't risk these getting out of sync.

Aside: yes, `id` is a terrible name for this value and we should fix it.  (My suggestion would be `terraName`.)  That's a *big* change so let's do it in a future refactoring.

Reply:
I checked the code for the Environments component, and the only thing we are passed is namespace and name, so we have to do our best with those values. If I were to continue working on this, I'd make a change to that component in Terra UI to pass more data so the implementor of `getLink` can make the decision about what is needed to send the user to the right location.

Reply:
hm, ok.  Is there an easy way for me to understand what the correct parameters for `getLink()` should be?

Reply:
optional suggestion: change the link to /analysis instead of /data

Reply:
Good point about `getLink()` parameters. I added [this comment](https://github.com/all-of-us/workbench/pull/8057/commits/a7ae39223ebdbddc188387faa5067593a2184b1c).